.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,44 @@
+name: Test Windows
+
+on:  [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: [python3]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ilammy/msvc-dev-cmd@v1
+    #- uses: warrenbuckley/setup-vswhere@v1
+    - uses: actions/setup-python@v2
+    - name: setup boost prerequisites
+      uses: lukka/run-vcpkg@v6
+      with:
+        #vcpkgArguments: boost:x64-windows
+        vcpkgGitCommitId: '8a9a97315aefb3f8bc5d81bf66ca0025938b9c91'
```

Comment:
Instead of using the static copy in vcpkg, wouldn't it make more sense to clone master/develop and use that? I'm definitely not a vcpkg expert, so I may be missing some of the benefit here.

Reply:
I assume by "static copy" you are referring to the fact that I'm fixing the commit ID, yes ? I tried to use the `lukka/run-vcpkg` action without specifying that, which prompted an error, instructing me to add the required git commit ID.

I don't really know what I'm doing here, just stumbling in the dark...

Reply:
By "static copy" I meant "always the same version of boost" e.g. 1.75.0. Since this is the development/master version of boost::python that is going into the next release, wouldn't it make sense to test with the version of boost that is also going into the next release?

Reply:
Not necessarily: If Boost.Python's code changes, it will be mostly its use of the Python C API, rather than its use of the rest of Boost. So the only reason to test against Boost develop would be to test that Boost itself doesn't break compatibility.
As far as Boost.Python is concerned, using a "static" version of Boost as prerequisite seems entirely reasonably and a good way to optimize CI builds.